providers/route53/route53Provider.go

```diff
@@ -305,3 +385,31 @@ func (r *route53Provider) EnsureDomainExists(domain string) error {
 	return err
 
 }
+
+func (r *route53Provider) waitUntilNameserversUpdate(operationId *string) error {
```

Comment:
This is a very good question. I definitely do not love the code required to wait, so I'd rather not. I'm curious about the exact semantics though. 

The update operation returns an "Operation ID"? Is that a commitment that they will make the change, or are there situations where it gets lost? If we are just waiting for them to update the tld nameservers, I think we could probably forego this. No other registrars really give real-time guarantees, and ttls on those records can be fairly long either way. But we do want to wait until we are highly confident it will go through in the future.

I'm torn. Thoughts @tlimoncelli ?